src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -303,7 +307,7 @@ public static void setNodeNameInContainerConfig(CreateContainerCmd containerConf
     /**
      * Retrieves the {@link Node} name chosen by
      * {@link #fillContainerConfig(CreateContainerCmd)}.
-     * 
+     *
```

Comment:
Unnecessary change - please undo. See the [contributing guide's coding style section](https://github.com/jenkinsci/docker-plugin/blob/master/CONTRIBUTING.md#submitting-pull-requests) for details.